guides/plugins/apps/app-scripts/cart-manipulation.md

```diff
@@ -87,6 +87,40 @@ Note that the state name should be unique, this means you should always use your
 
 In general, Showpare prices consist of gross and net prices and are currency dependent.  If you need price definitions in your app scripts (e.g. to add a absolute discount with a specific price) there are multiple ways to do so.
 
+### You can define price fields for custom field sets:
```

Comment:
Titles shouldn't be sentencens IMHO

what about 

```
### Price fields inside custom fields

You can define price fields for [custom field set](link-to-app-custom-field-docs).
{ code }
```

and something similiar for the config.xml